runner.py

```diff
@@ -34,15 +39,38 @@ def configure_gradio_ui_envs() -> None:
 
 
 if __name__ == "__main__":
+
+    cfg_file = os.environ.get("OLS_CONFIG_FILE", "olsconfig.yaml")
+
+    config.init_config(cfg_file)
+
+    configure_logging(config.ols_config.logging_config)
+    logger = logging.getLogger(__name__)
+    logger.info(f"Config loaded from {Path(cfg_file).resolve()}")
+
     configure_gradio_ui_envs()
 
     # NOTE: We import config here to avoid triggering import of anything
     # else via our code before other envs are set (mainly the gradio).
     from ols.utils import config
 
-    cfg_file = os.environ.get("OLS_CONFIG_FILE", "olsconfig.yaml")
-    config.init_config(cfg_file)
-
     configure_hugging_face_envs(config.ols_config)
+    config.init_query_filter()
+    config.init_vector_index()
+
+    host = "localhost" if config.dev_config.run_on_localhost else None
 
-    uvicorn.run("ols.app.main:app", host="127.0.0.1", port=8080, reload=True)
+    # use workers=1 so config loaded can be accessed from other modules
+    if config.dev_config.disable_tls:
+        # TLS is disabled, run without SSL configuration
+        uvicorn.run("ols.app.main:app", host=host, port=8080, workers=1)
+    else:
+        uvicorn.run(
+            "ols.app.main:app",
+            host=host,
+            port=8443,
```

Comment:
@xrajesh @raptorsun making sure you're aware of this....  with this PR, OLS is going to start listening on 8443 by default, unless tls is explicilty disabled in the config, so the OLS service needs to change accordingly.

@nemerna i think the best thing to do is first submit a PR to the ols operator that sets tls disabled=true.  Then we can merge this PR.  Then we can update the operator to setup the TLS/8443 service, certs, and set tls disabled=false.

Reply:
When this change is made, we need update[ the container port  here ](https://github.com/openshift/lightspeed-operator/blob/main/internal/controller/constants.go#L37). 
The service port below is for exposing the endpoint to other applications such as console.

Reply:
for now https://github.com/openshift/lightspeed-operator/pull/37 has the operator define the ols config w/ TLS disabled.

After this PR merges, the operator can be updated to properly configure TLS.